### DISCUSSION TITLE
nokov-send_extpose_quat

### DISCUSSION THREAD

**Discussion by Felix-kkk** [bitcraze/discussions#1652] [Q&A]

**Comment by gemenerik** [comment]
Hi! Can you clarify a bit; what are the two things that you expected to be the same but are not? The motion capture data and the Crazyflie's state estimate?

I’m not entirely sure what you’re asking in the second part either. Are you looking for guidance on how to use PositionHlCommander or high_level_commander to send global position commands after fixing the motion capture data issue? Or are you asking for confirmation that this is the correct approach? Could you clarify your goal here?

**Reply by Felix-kkk** [reply]
Hi!I reopened a [disscussion](https://github.com/orgs/bitcraze/discussions/1656) and would love to hear back from you,thank you!
